presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java

```diff
@@ -511,4 +511,15 @@ private static SqlVarbinary varbinary(int... bytesAsInts)
         }
         return new SqlVarbinary(bytes);
     }
+
+    @Test
+    public void testReverse()
+    {
+        assertFunction("REVERSE(CAST(null AS VARBINARY))", VARBINARY, sqlVarbinary(""));
```

Comment:
this one should probably be `assertFunction("REVERSE(CAST(null AS VARBINARY))", VARBINARY, null);` or something like that?

Reply:
Yup, thanks =) Sent another iter